omero-conda/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM ubuntu:14.04
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update -y && \
+        apt-get install -y --no-install-recommends \
```

Comment:
https://docs.docker.com/articles/dockerfile_best-practices/ says `apt-get upgrade` is bad, and `apt-get update` on a single line is bad (gets cached separately from the `install`.